ui/app/contexts/metametrics.js

```diff
@@ -18,14 +19,71 @@ import { getEnvironmentType } from '../../../app/scripts/lib/util'
 import {
   sendMetaMetricsEvent,
   sendCountIsTrackable,
+  mixpanelId,
 } from '../helpers/utils/metametrics.util'
 
+const analytics = new Analytics(mixpanelId)
```

Comment:
The [Segment documentation suggests](https://segment.com/docs/connections/sources/catalog/libraries/server/node/#development) that we use the `{ flushAt: 1 }` property when in development